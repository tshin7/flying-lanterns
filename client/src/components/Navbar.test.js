import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './Navbar';

describe('Navbar', () => {
  it('should render correctly with no props', () => {
    const component = shallow(<Navbar />);

    expect(component).toMatchSnapshot();
  });
});
