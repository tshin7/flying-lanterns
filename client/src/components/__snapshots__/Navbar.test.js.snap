// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render correctly with no props 1`] = `
<Route>
  <Component />
</Route>
`;
